### PR TITLE
Integration Shim Tests improvements

### PR DESCRIPTION
**Pull request by tsvetomir-nedyalkov**
#### Description
Tests improvements, needed for the CI implementation and for running the tests on older Cordova Webview

#### Changes
1. The tests are executing against different apps
2. Added fetch polyfill
3. The tests are transpiled with Babel

### CODE REVIEW

**Comment by thomasconner** [comment]
@georgiwe I don't see any reason not to approved this PR. I see there are no comments. Can you approve it if you agree?

**Comment by georgiwe** [comment]
I and Tsvetomir talk offline. He's still working out some issues and has asked me to ignore the PR for now. I believe we might finish things up today.

File: .gitignore [hunk 1]
```diff
@@ -33,3 +33,6 @@ packages/kinvey-html5-sdk/test/index.html
 #generated test apps
 packages/kinvey-nativescript-sdk/TestApp/
 packages/kinvey-phonegap-sdk/KinveyCordovaTestApp/
+packages/kinvey-phonegap-sdk/test/config.js
+packages/kinvey-nativescript-sdk/test/config.js
```
[the remark this answers is not in the document]
Comment (tsvetomir-nedyalkov) [reply .gitignore]: These are the result configs, which are auto generated from the base configs, so I added them in gitignore...

File: packages/kinvey-html5-sdk/test/tasks/webRunTests.js [hunk 2]
```diff
@@ -16,8 +16,7 @@ const webRunTests = (staticPort, runner) =>
         } else {
             const chrome = spawnHeadlessChromium(args);
             chrome.stderr.on('data', d => reject(d.toString()));
-            chrome.on('close', () => resolve());
-            runner.on('log.end', resolve);
+            resolve();
```
[the remark this answers is not in the document]
Comment (tsvetomir-nedyalkov) [reply packages/kinvey-html5-sdk/test/tasks/webRunTests.js]: I have logged the issue - https://kinvey.atlassian.net/browse/QA-157

**Pull request merged**